lib/stack_master/validator.rb

```diff
@@ -7,11 +7,12 @@ def initialize(stack_definition)
     end
 
     def perform
+      StackMaster.stdout.print "#{@stack_definition.stack_name}: "
```

Comment:
It's only because this command can output it's result in one line that a print makes sense. It probably wouldn't make sense for apply which does a lot of things.